packages/apollo/src/index.ts

```diff
@@ -1,14 +1,17 @@
 import { ApolloLogger } from './apollo';
-import { Options } from './types';
+import { Options, GraphQlLogEntry } from './types';
 
-export { ApolloLogger, Options };
-
-export function apolloLogger(options?: Options): ApolloLogger {
-  return new ApolloLogger(options);
-}
+export { ApolloLogger, Options, GraphQlLogEntry };
 
 declare module '@debugr/core' {
-  export interface Plugins {
-    apollo: ApolloLogger;
+  export interface TContextFixed {
+    queryName?: string;
+  }
```

Comment:
k čemu je `TContextFixed`?

Reply:
jo... Ja potreboval to udelat jako interface a zaroven to mit jako typ :D. Tak z tohoto dedim do typu a toto pripadne rozsiruju o dalsi data